features/support/api_helpers.rb

```diff
@@ -267,12 +267,33 @@ def create_supplier(custom_supplier_data = {})
     }]
   }
   supplier_data.update(custom_supplier_data)
+
+  creatable_attrs = {}
+  updatable_attrs = {}
+  supplier_data.each do |key, value|
+    if %i[id companiesHouseNumber contactInformation dunsNumber description name].include? key
```

Comment:
A one line comment here about where the restriction lies might be useful (could be overkill, but with func tests we usually need all the help we can get). `The API's validation restricts supplier creation to the following subset of attributes, any other attributes must be added in a second update`. It is the validation, right - not at model level?

Reply:
Yeah it's in the json schema